datahub-web-react/src/app/homeV3/context/hooks/useShowHomePageRedesign.ts

```diff
@@ -0,0 +1,31 @@
+import { useAppConfig } from '@app/useAppConfig';
+
+export function useShowHomePageRedesign() {
+    const appConfig = useAppConfig();
+    const { showHomePageRedesign } = appConfig.config.featureFlags;
```

Comment:
:( naming wise, now we have so many redesign, v2, v3 etc. it's okay. we can remove later.